lib/constructor.js

```diff
@@ -283,8 +283,8 @@ const Sharp = function (input, options) {
     tileCentre: false,
     tileId: 'https://example.com/iiif',
     timeoutSeconds: 0,
-    linearA: 1,
-    linearB: 0,
+    linearA: [1.0],
```

Comment:
We could default these to an empty array, which should simplify the "should we perform a linear operation?" logic in the C++.